MoneySplitter.Services/Api/ApiUrlBuilder.cs

```diff
@@ -31,20 +31,24 @@ public Uri SearchUsers(string query)
                 query));
         }
 
-        public Uri AddFriend()
+        public Uri AddFriend(int friendId)
         {
             return new Uri(string.Concat(
                 Defines.Api.WEB_API_URL,
                 Defines.Api.Friends.FRIENDS,
-                Defines.Api.Friends.ADD_FRIEND));
+                Defines.Api.Friends.ADD_FRIEND,
+                Defines.Api.Friends.FRIEND_ID,
+                friendId.ToString()));
         }
 
-        public Uri RemoveFriend()
+        public Uri RemoveFriend(int friendId)
         {
             return new Uri(string.Concat(
                 Defines.Api.WEB_API_URL,
                 Defines.Api.Friends.FRIENDS,
-                Defines.Api.Friends.REMOVE_FRIEND));
+                Defines.Api.Friends.REMOVE_FRIEND,
+                Defines.Api.Friends.FRIEND_ID,
+                friendId.ToString()));
```

Comment:
As I know you don't need call ToString()

Reply:
remove toString